app/views/admin/infobase/_orders.html.haml

```diff
@@ -0,0 +1,17 @@
+.ui.top.attached.tabular.menu
+  %a.item{"data-tab": "Monday"} Monday
+  %a.item{"data-tab": "Tuesday"} Tuesday
+  %a.item{"data-tab": "Wednesday"} Wednesday
+  %a.item{"data-tab": "Thursday"} Thursday
+  %a.item{"data-tab": "Friday"} Friday
+
+.ui.bottom.attached.tab.segment{"data-tab": "Monday", id: 'order-tab-attached'}
```

Comment:
`We are not paid for extra lines` - you better remember this

loop and `orders_for_XX` should be `orders_for(XX)`